### PR TITLE
setup sqlite web interface with log volume

### PR DESCRIPTION
**Pull request by RamonSaboya**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -85,12 +85,25 @@ services:
       ME_CONFIG_MONGODB_ADMINPASSWORD: vitaes
     networks:
     - net
+  sqlite-web:
+    image: coleifer/sqlite-web
+    restart: always
+    ports:
```
Comment (Arthurlpgc) [inline docker-compose.yml]: 👀

Comment (RamonSaboya) [reply docker-compose.yml]: nothing to see here

**Comment by Arthurlpgc** [comment]
Completely crazy ports, but fine otherwise

**Comment by Arthurlpgc** [comment]
i

**Pull request merged**